proposal.md

```diff
@@ -0,0 +1,17 @@
+# Proposal
+
+## Motivation and Purpose
+
+Our role: Tsunami data science enthusiasts
+
+Target Audience: Fellow tsunami enthusiasts
+
+Vancouver sits on the edge of the Pacific Rim of Fire - a region where most of the volcanic eruptions and earthquakes occur. These eruptions occur in the middle of the Pacific Ocean and travel towards land as tsunamis. Each vary in magnitude and location, and cause physical and economic damage. How do we track each tsunami that has occured? How do we know how the magnitude of tsunamis change over time? How can we compare the damage of each tsunami to another in the same year or between countries?
+
+To address these questions, we propose building a data visualisation app that allows these enthusiasts to visually explore a dataset of past tsunami instances over the past few hundred years (the data set consist of tsunamis from 2100BC, but we have decided to limit it to 1800AD). Our app will show how the distribution of tsunamis vary over time, and users can use the tooltip to find our more information about each tsunami. It will also list the top 10 tsunamis that occur in the given time period that the use has selected.
```

Comment:
```suggestion
To address these questions, we propose building a data visualization app that allows these enthusiasts to visually explore a dataset of past tsunami instances over the past few hundred years (the data set consists of tsunamis from 2100BC, but we have decided to limit it to 1800AD). Our app will show how the distribution of tsunamis varies over time, and users can use the tooltip to find out more information about each tsunami. It will also list the top 10 tsunamis that occur in the given time period that the user has selected.
```